.env.sample

```diff
@@ -0,0 +1,4 @@
+AGORA_FILES=files
+AGORA_PORT=8080
+LND_RPC_AUTHORITY=
+LND_DIR=.lnd
```

Comment:
Need to add LND enabled var or change entry point

Reply:
I've tried out a few solutions. I think the best option is to hardcode the content directory as `files`, which matches the current URL structure. And the LND `tls.cert` and `invoices.macaroon` will be in the `.lnd` folder.